recipes/libxml2/all/test_cmake_module_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package C)
+
+find_package(LibXml2 REQUIRED)
+
+message("LIBXML2_FOUND: ${LIBXML2_FOUND}")
+message("LIBXML2_INCLUDE_DIR: ${LIBXML2_INCLUDE_DIR}")
+message("LIBXML2_INCLUDE_DIRS: ${LIBXML2_INCLUDE_DIRS}")
+message("LIBXML2_LIBRARIES: ${LIBXML2_LIBRARIES}")
+message("LIBXML2_LIBRARY: ${LIBXML2_LIBRARY}")
+message("LIBXML2_DEFINITIONS: ${LIBXML2_DEFINITIONS}")
+message("LIBXML2_VERSION_STRING: ${LIBXML2_VERSION_STRING}")
```

Comment:
it's not a test

Reply:
It was there before, and things that use it as a module tend to want these defined,
so I was checking with these.
Happy for them to be removed in a future PR :)